packages/caliper-samples/src/contract/scalardl/src/main/java/com/example/contract/simple/SimpleOpen.java

```diff
@@ -0,0 +1,26 @@
+package com.example.contract.simple;
+
+import com.scalar.ledger.asset.Asset;
+import com.scalar.ledger.contract.Contract;
+import com.scalar.ledger.ledger.Ledger;
+import java.util.Optional;
+import javax.json.Json;
+import javax.json.JsonObject;
+
+public class SimpleOpen extends Contract {
+
+  @Override
+  public JsonObject invoke(Ledger ledger, JsonObject argument, Optional<JsonObject> properties) {
+    String accountId = argument.getString("account");
+    int balance = argument.getInt("money");
+
+    Optional<Asset> asset = ledger.get(accountId);
+
+    if (!asset.isPresent()) {
+      ledger.put(accountId, Json.createObjectBuilder().add("balance", balance).build());
+    }
+    // Should write error handling for case that account already exists
```

Comment:
Please do it !
Please see the doc for how to do it properly.
https://github.com/scalar-labs/scalardl/blob/master/docs/how-to-write-contract.md

Reply:
Done.